woodwork/tests/utils/test_utils.py

```diff
@@ -456,6 +456,7 @@ def test_col_is_datetime():
         pd.Series(["a", "b", "c"]),
         pd.Series([pd.NA, "b", "c"]),
         pd.Series([pd.NA, pd.NA, pd.NA]),
+        pd.Series(["12/01/2022", pd.NA, pd.NA]),
```

Comment:
Minor detail but this situation is already covered by line 455, so this can be removed. Didn't realize this was already being tested when I asked about handling of this type of series previously.

Reply:
removed